documentation/design/iot/README.adoc

```diff
@@ -1,38 +1,148 @@
+:icons: font
 
 = IoT support
 
+[.lead]
 This document describes how IoT is added into the EnMasse system.
 
+NOTE: This is the updated version for EnMasse Core v1.
+
 == Overview
 
-EnMasse IoT support is an additional layer on top of the EnMasse system. It supports IoT
-specific use cases and protocols, like "telemetry ingestion", "command & control", "MQTT", "HTTP", ...
+EnMasse IoT support is an additional layer on top of the
+EnMasse system. It supports IoT specific use cases and
+protocols, like "telemetry ingestion", "command & control",
+"MQTT", "HTTP", …
 
 This is implemented based on the Eclipse Hono project.
 
+== IoT Config CRD
+
+[.lead]
+An "IoT Config" is an IoT infrastructure which can serve multiple IoT tenants (IoTProject).
+
+An IoT Config:
+
+* Is part of the EnMasse infrastructure namespace.
+* Is configured to bind to exactly one _messaging infrastructure_.
+* Has the name `default`.
+* Can currently only exist once in the cluster.
++
+NOTE: The goal is to lift this limitation in a future version and
+allow for multiple IoT infrastructure instances. Each IoT infrastructure
+pointing to exactly one messaging infrastructure.
+* Serves all IoT projects assigned to the IoT infrastructure.
++
+This is an implicit assignment. An IoT project resides in a Kubernetes
+namespace, which is assigned to a messaging tenant, which is assigned
+to a messaging infrastructure.
+
+*Target persona:* IoT service admin
+
+*Proposal:* Change the name from `IoTConfig` to `IoTInfrastructure`
```

Comment:
Agreed on renaming it, just noting we should either call it IoTInfra or rename MessagingInfra to MessagingInfrastructure

Reply:
I would propose then to go for the full name. I don't think it s very common to abbreviate in the Kubernetes world.

Reply:
anyway add shortnames please into CRDs :)